examples/policies/host/lock-down-dev-vms.yaml

```diff
@@ -0,0 +1,119 @@
+apiVersion: "cilium.io/v2"
```

Comment:
Is this example automagically picked up by the CI somehow or were you testing this just locally?

Would be awesome to hook it in as part of regular tests to prevent breakage. (But I also recognize that could involve more changes which may/may not make sense for backport).

Reply:
I guess this question is largely already addressed by your PR description above so I'll await your followups :-)